src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        final int limitnumber = 100;

```

Comment:
It's not a constant, google what a constant is. [Also here's what they should be called](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)

Reply:
i have a quastion about constant. if i print:

final int NUMBER = random.nextInt(100);

will it be constat? If i will use static final, i have mistake

Reply:
if i make 100 constant out of mathod getRandomBall(), i will get i Maven mistake at Git. If i make 
static final int NUMBER = random.nextInt(100); 
then code will not compilated in java, becouse i can not use static in method getRandomBall(). Should i make it just 
final int NUMBER = random.nextInt(100); ?